.github/workflows/deploy-dev.yml

```diff
@@ -34,6 +39,19 @@ jobs:
           chmod +x changelog.sh
           ./changelog.sh
 
+      - name: Stage Changes
+        run: git add .
+        
+      - name: Commit Changes
+        run: |
+          git commit -m "chore(release): publish version <tag> in wordpress.org"
```

Comment:
Instead of `<tag>`, use the number of the tag being published

Reply:
Added the code, could you please check and let me know if anything remains?